src/common/front-end-routing/verify-that-all-routes-have-route-component.test.ts

```diff
@@ -9,15 +9,17 @@ import { filter, map, matches } from "lodash/fp";
 import clusterStoreInjectable from "../cluster-store/cluster-store.injectable";
 import type { ClusterStore } from "../cluster-store/cluster-store";
 import { pipeline } from "@ogre-tools/fp";
+import createStoresAndApisInjectable from "../../renderer/create-stores-apis.injectable";
 
 describe("verify-that-all-routes-have-component", () => {
   it("verify that routes have route component", async () => {
     const rendererDi = getDiForUnitTesting({ doGeneralOverrides: true });
 
-    rendererDi.override(
-      clusterStoreInjectable,
-      () => ({ getById: (): null => null } as unknown as ClusterStore),
-    );
+    rendererDi.override(createStoresAndApisInjectable, () => true);
+
+    rendererDi.override(clusterStoreInjectable, () => ({
+      getById: () => null,
+    } as unknown as ClusterStore));
```

Comment:
Note: this casting is damage caused by breached Interface Segregation Principle somewhere else. Without the breach, the override could be for something specific, instead of this big ball of everything. 

Done right, typing would be more robust, and the test would be simpler and less brittle.

Reply:
Oh I completely agree. There should much more use of the (already existsing) `getPodById` injectable. Much easier to test around.